opm/autodiff/StandardWells_impl.hpp

```diff
@@ -365,6 +374,7 @@ namespace Opm
             return;
         } else {
             const std::vector<int>& well_cells = wellOps().well_cells;
+            const int num_phases_ = wells().number_of_phases;
```

Comment:
The variable name should not end with an underscore, even when that causes a few more lines below to require the change, too.